src/display/annotation_layer.js

```diff
@@ -2847,14 +2766,10 @@ class FileAttachmentAnnotationElement extends AnnotationElement {
         }
       }
     }
-    trigger.classList.add("popupTriggerArea");
```

Comment:
Given the failure in test `issue16800-page2` I'm guessing that my suggestion to remove this line was probably wrong, sorry about that!